src/vm/eventpipebuffermanager.cpp

```diff
@@ -10,6 +10,13 @@
 
 #ifdef FEATURE_PERFTRACING
 
+#ifndef MIN
```

Comment:
nit: Ideally we should use `std::min` and `std::max` (maybe not available in clr)

Reply:
There is a macro USE_STL, but it is only defined in the PAL. Elsewhere the runtime avoids STL usage. This avoidance may be historical and no longer have a sound basis, but digging into that investigation probably isn't worth it solely for simple functions like min and max. Looking around I did find a Min/Max implemented in clrtypes.h so I got rid of these defines and converged on those definitions instead.